src/helix-ui/elements/HXFileInputElement.less

```diff
@@ -24,14 +24,23 @@
   }
 
   #hxLabel {
-    margin-left: 0.5rem;
-
     &:empty {
       display: none;
     }
   }
 }
 
+:host(:not([icon])) {
```

Comment:
Similar comment as above... 

```less
:host {
  // styles without icon
}
:host([icon]) {
  // styles WITH icon
}
```

(it's possible reconfiguring it this way may remove the need for `!important` below)